doc/source/features/pxc-strict-mode.rst

```diff
@@ -368,8 +376,9 @@ the following happens:
 
  At runtime, any CTAS operation is denied and an error is logged.
 
-.. note:: CTAS operations for temporary tables are permitted
-   even in strict mode.
+.. note::
```

Comment:
this is not true. Old behavior is retained. Let me clarify the comment I made in the PXC ticket.
Sorry for the confusion but why I meant was temporary table should not be used as SOURCE table in CTAS as the temporary table are not present on all the nodes.